robusta_krr/core/models/result.py

```diff
@@ -22,26 +21,13 @@ class ResourceRecommendation(pd.BaseModel):
     info: dict[ResourceType, Optional[str]]
 
 
-class Metric(pd.BaseModel):
-    query: str
-    start_time: datetime
-    end_time: datetime
-    step: str
-
-
-MetricsData = dict[ResourceType, Metric]
-
-
 class ResourceScan(pd.BaseModel):
     object: K8sObjectData
     recommended: ResourceRecommendation
     severity: Severity
-    metrics: MetricsData
```

Comment:
i think this changes were needed to display the queries used in the json output so that we could eventually make graphs in frontend.
I dont see the queries now when i run it with -f json

Reply:
We discussed that with @pablos44 and @yuryrudey 
Frontend will use it's metrics